CouchDocument/AccessToken.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the FOSOAuthServerBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\OAuthServerBundle\CouchDocument;
```

Comment:
Should we move MongoDB to make way for CouchDB? This way, one of them seems "more important".

Reply:
I think Apache CouchDB can have real advantage. For my project, I use it for scalability, and master-master replication capability. But, like you said, library are not up-to-date, I try to make some PR to keep these libraries usable with Symfony 3.X but they are not ready for 4

Reply:
I meant, should we move the MongoDB models to say `FOS\Document\MongoDB` to make room for other ODM implementations?